### DISCUSSION TITLE
How to Handle Promised Options or Promised Securities?

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#94] [Suggestions] [answered]
Most platforms let you create drafts of new securities. Also, many other securities will require that you include promised options in pre-money capitalization. We should think about ways to handle this.

**Comment by JSv4** [comment, marked as answer]
**TWG Notes 2/15/22:**

- This could potentially pull double duty with "draft securities", even though **the concept of a draft isn't really designed to be a "promised" security**. 
  - Nevertheless, seems like this is how a lot of people handle this. 
  - Some systems won't export draft securities, however, which creates compatibility issues. 
  - Also many types of actions that are potentially possible with promised securities are not permissible with drafts in the various systems.
- Could be a boolean flag in the issuance object. 
- **For version 1, this is out of scope.**

**Comment by strichter** [comment]
Shoobx tracks promises separately in its own accounts. That allows one to consider the grants in the captable and also help the user "remember" the grants upon granting.

**Reply by JSv4** [reply]
Interesting @strichter. Thanks for adding to the conversation! Just to make sure I understand the approach you guys use, are you creating a whole separate ledger with only promised securities or do you assign a phantom/system owner to certain promised securities and add them to the ledger, letting the system filter out securities with that dummy owner as appropriate? Something else?

**Comment by strichter** [comment]
We basically create a separate ledger. We can then decide whether to include the promise ledger in a cap table or not based on the situation. We also use the promise ledger to determine whether enough shares remain in a SIP. And a separate ledger is what I would expect for the purposes of the OCF format.

(BTW, we call ledgers accounts as we use a double accounting system to ensure integrity. So a promises are tracked in a promise account for a specific SIP subtracting shares from the available shares account of the pool. This is of course highly simplified to what's actually going on, but you get the idea.)

**Reply by JSv4** [reply]
Interesting. So, with your approach, we wouldn't need to create a specific object type or property at all. You could have two OCF-compatible event stacks and move the promised securities over to the "official" event log when and as appropriate?

**Comment by strichter** [comment]
That would work just fine. For promises we really just have to track when was the promise made for what amount/security.

**Reply by JSv4** [reply]
Nice. LOVE this solution. Thanks for suggesting it!
